.markdownlint.yaml

```diff
@@ -31,3 +31,8 @@ MD024:
 #
 # https://github.com/DavidAnson/markdownlint/blob/main/doc/md051.md
 MD051: false
+
+# for blog posts
```

Comment:
Would be A+ if you could include the rule name and possible link to the rule docs for each; makes it a lot easier to grok and maintain in the long run